k4FWCore/src/PodioDataSvc.cpp

```diff
@@ -43,10 +44,10 @@ StatusCode PodioDataSvc::initialize() {
     if (m_filenames[0] != "") {
       m_reading_from_file = true;
       m_reader.openFiles(m_filenames);
-      m_eventMax = m_reader.getEntries("events");
+      m_numAvailableEvents = m_reader.getEntries("events");
 
       if (m_1stEvtEntry != 0) {
-        m_eventMax -= m_1stEvtEntry;
+        m_numAvailableEvents -= m_1stEvtEntry;
```

Comment:
This `if` condition has no effect, since `-= 0` will also leave `m_numAvailableEvents` unchanged.